cms/static/js/views/settings/grading.js

```diff
@@ -96,7 +102,13 @@ var GradingView = ValidatingView.extend({
         };
         var grace_period = this.model.get('grace_period');
         this.$el.find('#course-grading-graceperiod').val(
-            format(grace_period.hours) + ':' + format(grace_period.minutes)
+            StringUtils.interpolate(
```

Comment:
Ok, copy. Reverting it back to
`format(grace_period.hours) + ':' + format(grace_period.minutes)`